src/vorta/views/repo_add_dialog.py

```diff
@@ -27,7 +30,7 @@ def __init__(self, parent=None):
         self.saveButton = self.buttonBox.button(QDialogButtonBox.StandardButton.Ok)
         self.saveButton.setText(self.tr("Add"))
 
-        self.buttonBox.rejected.connect(self.close)
+        self.buttonBox.rejected.connect(lambda: (self.cancel_job(), self.close()))
```

Comment:
This lambda is getting out of hand. Can you move the code into a method (e.g. `cancel_job` which would have to be renamed in that case)?

Reply:
Didn't get that actually 😅 , the `cancel_job` is already a method 🤔